src/Client/ClientBase.cpp

```diff
@@ -3008,24 +3003,6 @@ void ClientBase::init(int argc, char ** argv)
 
     addOptions(options_description);
 
-    auto getter = [](const auto & op)
-    {
-        String op_long_name = op->long_name();
-        return "--" + String(op_long_name);
-    };
-
-    if (options_description.main_description)
-    {
-        const auto & main_options = options_description.main_description->options();
-        std::transform(main_options.begin(), main_options.end(), std::back_inserter(cmd_options), getter);
-    }
-
-    if (options_description.external_description)
-    {
-        const auto & external_options = options_description.external_description->options();
-        std::transform(external_options.begin(), external_options.end(), std::back_inserter(cmd_options), getter);
-    }
-
```

Comment:
Consider to add another option (something like `--verbose` or `-vvv`) to display all the available settings.